src/server/gc_worker/gc_worker.rs

```diff
@@ -1062,6 +1074,37 @@ impl<E: Engine> Runnable for GcRunner<E> {
     }
 }
 
+impl<E: Engine> GcRunner<E> {
+    pub fn new(
+        store_id: u64,
+        engine: E,
+        flow_info_sender: Sender<FlowInfo>,
+        cfg_tracker: Tracker<GcConfig>,
+        cfg: GcConfig,
+        pool: Remote<TaskCell>,
+    ) -> Self {
+        Self {
+            inner: GcRunnerCore::new(store_id, engine, flow_info_sender, cfg_tracker, cfg),
+            pool,
+        }
+    }
+}
+
+impl<E: Engine> Runnable for GcRunner<E> {
+    type Task = GcTask<E::Local>;
+
+    #[inline]
+    fn run(&mut self, task: GcTask<E::Local>) {
+        // Refresh config before handle task
+        self.inner.refresh_cfg();
+
+        let mut inner = self.inner.clone();
+        self.pool.spawn(async move {
```

Comment:
Why not make the Worker itself support multiple threads, then we don't need to  a worker just to spawn task

Reply:
here refresh_cfg only needs to be done once.
And worker itself is hard to be multi-threads without major code change.  And not all runner can run in multi-threaded.

Reply:
okay, let's leave it to future PR.